openshift/buildConfig-template.yaml

```diff
@@ -57,13 +57,13 @@ parameters:
   displayName: Thoth Kebechet git repository
   required: true
   name: GITHUB_URL
-  value: 'https://github.com/thoth-station/kebechet'
+  value: 'https://github.com/KPostOffice/kebechet'
```

Comment:
Please revert this back so we can merge the correct URL (also for reference bellow).